penquins/penquins.py

```diff
@@ -29,6 +36,31 @@
 DEFAULT_BACKOFF_FACTOR: int = 1
 
 
+def get_cones(path, cumprob):
+    max_order = None
+    with fits.open(path) as hdul:
+        hdul[1].columns
```

Comment:
What does this do?

Reply:
Basically, there is no (at least to my knowledge) parameters to open a moc skymap with mocpy that allows me to keep the Nth most probably region. So first I open it with astropy.fits to get uniq, prob and max_order which together with the cumprob can be used with one of mocpy's methods to generate a moc skymap using a cumprob value.

Reply:
the with open is simply so that I use a context manager to avoid keeping the file in memory throughout the rest of the function.

Reply:
ah ok.